apps/web/src/pages/index.tsx

```diff
@@ -1,7 +1,15 @@
 import type { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { useEffect } from 'react';
+
+import { PageLoading } from '@/components/loading';
 
 const Home: NextPage = () => {
-  return <div title="Home Page"></div>;
+  const router = useRouter();
+  useEffect(() => {
+    router.replace('/workspace');
+  }, [router]);
+  return <PageLoading />;
 };
 
 export default Home;
```

Comment:
Since Home does not require client logic, we can use next js redirect instead:

```suggestion
const Home: NextPage = () => {
  return <PageLoading />;
};

export default Home;

export function getStaticProps() {
  return {
    redirect: {
      destination: '/workspace',
      permanent: false,
    },
  }
}
```

Reply:
Dont do this, `next export` doesn't support this

Reply:
See https://github.com/toeverything/AFFiNE/actions/runs/4226181756/jobs/7339338840